src/studio/src/designer/frontend/ux-editor/utils/urlHelper.ts

```diff
@@ -1,80 +1,26 @@
-import type { IAltinnWindow } from '../types/global';
-
-const altinnWindow: IAltinnWindow = window as Window as IAltinnWindow;
-const basePath = `${altinnWindow.location.origin}/designer/${altinnWindow.org}/${altinnWindow.app}`;
-
-export const getFetchFormLayoutUrl = (): string => {
-  return `${basePath}/UIEditor/GetFormLayout`;
-};
-
-export const getSaveFormLayoutUrl = (layoutName: string): string => {
-  return `${basePath}/UIEditor/SaveFormLayout/${layoutName}`;
-};
-
-export const getUpdateFormLayoutNameUrl = (layoutName: string): string => {
-  return `${basePath}/UIEditor/UpdateFormLayoutName/${layoutName}`;
-};
-
-export const getDeleteForLayoutUrl = (layout: string): string => {
-  return `${basePath}/UIEditor/DeleteFormLayout/${layout}`;
-};
-
-export const getLayoutSettingsUrl = (): string => {
-  return `${basePath}/UIEditor/GetLayoutSettings`;
-};
-
-export const getSaveLayoutSettingsUrl = (): string => {
-  return `${basePath}/UIEditor/SaveLayoutSettings`;
-};
-
-export const getSaveServiceConfigurationUrl = (): string => {
-  return `${basePath}/UIEditor/SaveJsonFile?fileName=RuleConfiguration.json`;
-};
-
-export const getAddApplicationMetadataUrl = (): string => {
-  return `${basePath}/UIEditor/AddMetadataForAttachment`;
-};
-
-export const getDeleteApplicationMetadataUrl = (): string => {
-  return `${basePath}/UIEditor/DeleteMetadataForAttachment?id=`;
-};
-
-export const getUpdateApplicationMetadataUrl = (): string => {
-  return `${basePath}/UIEditor/UpdateMetadataForAttachment`;
-};
-
-export const getLayoutSettingsSchemaUrl = (): string => {
-  return 'https://altinncdn.no/schemas/json/layout/layoutSettings.schema.v1.json';
-};
-
-export const getLayoutSchemaUrl = (): string => {
-  return 'https://altinncdn.no/schemas/json/layout/layout.schema.v1.json';
-};
-
-export const getAddTextResourcesUrl = (): string => {
-  return `${basePath}/UIEditor/AddTextResources`;
-};
-
-export const getWidgetsSettingsUrl = (): string => {
-  return `${basePath}/UIEditor/GetWidgetSettings`;
-};
-
-export const getLoadTextResourcesUrl = (languageCode: string) => {
-  return `${basePath}/UIEditor/GetTextResources/${languageCode}`;
-};
-
-export const getFetchDataModelUrl = () => {
-  return `${basePath}/Model/GetJson`;
-};
-
-export const getFetchLanguageUrl = (languageCode: string) => {
-  return `${window.location.origin}/designer/frontend/lang/${languageCode}.json`;
-};
-
-export const getFetchRuleModelUrl = () => {
-  return `${basePath}/UIEditor/GetRuleHandler`;
-};
-
-export const getFetchRuleConfigurationUrl = () => {
-  return `${basePath}/UIEditor/GetJsonFile?fileName=RuleConfiguration.json`;
-};
+import { _useParamsClassCompHack } from 'app-shared/utils/_useParamsClassCompHack';
+
+const { org, app } = _useParamsClassCompHack();
```

Comment:
interesting name :)

Reply:
Its temporary. Its to emulate `useParams` in class components... It also need to be prefixed because `use` is a reserved prefix in the world of React.

Reply:
I see, but it would be a hook regardless right?